kolibri/plugins/management/assets/src/views/user-page/reset-user-password-modal.vue

```diff
@@ -0,0 +1,165 @@
+
```

Comment:
remove these newlines here

Reply:
Wow, how embarrassing.

Reply:
@jonboiser Updated.

Reply:
They're still here :ghost:!

Reply:
It looks like it, but see changes here: https://github.com/learningequality/kolibri/pull/2154/commits/b7d4726c8d9d7368d4f13e3e66a71511dd7ce089

Reply:
Hmm, something weird is going on. The spaces are still here on the PR overview, but they seem gone when you look at the diff.